src/utils/mp_configs_verifyer/configs_dump_verifyer.h

```diff
@@ -21,8 +21,7 @@ class configs_verifyer
     bool const verify(u8* data, u32 data_size, string256& diff);
 
 private:
-    typedef u8 sha_checksum_t[crypto::xr_sha256::digest_length];
-    bool const verify_dsign(u8* data, u32 data_size, sha_checksum_t& sha_checksum);
+    bool const verify_dsign(u8* data, u32 data_size, crypto::xr_sha1::hash_t& sha_checksum);
```

Comment:
u8* -> const u8* const data

Reply:
Will do

Reply:
I decided not to change this part of code, because it is from another project.